logger/handler/ringbuffer_test.go

```diff
@@ -0,0 +1,43 @@
+package handler
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/deis/deis/logger/syslog"
+)
+
+func TestGetProjectName(t *testing.T) {
+	pname, err := getProjectName("junk test[junk] junk junk junk")
+	if err != nil {
+		t.Errorf("Failed to get project name: %s", err)
+	}
+
+	if pname != "test" {
+		t.Errorf("Expected project name 'test', got '%s'", pname)
+	}
+
+}
```

Comment:
maybe this is better re-written like so (more idiomatic when it comes to testing style)

```
func TestGetProjectName(t *testing.T) {
	testStr := "junk test[junk] junk junk junk"
	got, err := getProjectName(testStr)
	if err != nil {
		t.Fatalf("getProjectName(%q) got an expected error: %s", testStr, err)
	}

	if want := "test"; want != got {
		t.Errorf("getProjectName(%q): want %s got %s", want, got)
	}
}
```

So basically, there's no need to continue if there was an error, also the format of the message is far more interesting if it's function(args) and return. Also I know this isn't really Go style specific but I personally prefer not to see empty line right after `{` or right before `}` feels like a wasted space on my screen :trollface: